python/notebooks/03-semantic-function-inline.ipynb

```diff
@@ -84,7 +84,7 @@
     "    kernel.add_text_completion_service(\"dv\", azure_text_service)\n",
     "else:\n",
     "    api_key, org_id = sk.openai_settings_from_dot_env()\n",
-    "    oai_text_service = OpenAITextCompletion(ai_model_id=\"text-davinci-003\", api_key=api_key, org_id=org_id)\n",
+    "    oai_text_service = OpenAITextCompletion(ai_model_id=\"gpt-4\", api_key=api_key, org_id=org_id)\n",
```

Comment:
either use gpt-35-turbo-instruct (a text model) or change the class to OpenAIChatCompletion, probably the first.

Reply:
Good catch, thanks. Was testing something, and realized gpt-4 can't be used as a text completion model. :) Forgot to update it.